docs/opentsdb-subset.yaml

```diff
@@ -0,0 +1,319 @@
+    post:
+      parameters:
+        - in: header
+          name: X-Tenant
+          required: false
+          description: Optional/configurable, tenant originated observed metric value
+        - in: query
+          name: summary
+          required: false
+          schema:
+            type: boolean
+          description: |
+            When present with empty value or "true", repsonse includes summary of metrics written.
+        - in: query
+          name: details
+          schema:
+            type: boolean
+          description: |
+            When present with empty value or "true", response includes summary of metrics written
+            and errors.
+          required: false
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              anyOf:
+                - $ref: '#/components/schemas/Metric'
+                - type: array
+                  items:
+                    $ref: '#/components/schemas/Metric'
+            examples:
+              single:
+                value: |
+                  {
+                    "metric": "memory_free",
+                    "tags": {
+                      "tenant": "t-1",
+                      "os": "linux",
+                      "host": "h-1",
+                      "deployment": "prod"
+                    },
+                    "timestamp": 1604514466,
+                    "value": 123
+                  }
+              multi:
+                value: |
+                  [
+                    {
+                      "metric": "memory_free",
+                      "tags": {
+                        "tenant": "t-1",
+                        "os": "linux",
+                        "host": "h-1",
+                        "deployment": "prod"
+                      },
+                      "timestamp": 1604514466,
+                      "value": 123
+                    },
+                    {
+                      "metric": "cpu_usage",
+                      "tags": {
+                        "tenant": "t-2",
+                        "os": "linux",
+                        "host": "host-b",
+                        "deployment": "prod"
+                      },
+                      "timestamp": 1604514539,
+                      "value": 5.0
+                    }
+                  ]
+      responses:
+        204:
+          description: All data points were stored successfully
+        200:
+          description: |
+            When summary or details parameter is specified response includes the
+            corresponding content
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PutResponse'
+              examples:
+                summary:
+                  value: |
+                    {
+                      "failed": 0,
+                      "success": 5
+                    }
+                detailsWithError:
```

Comment:
what is the difference between "detailsWithError"  and the "detailsWithErrors" below?

Reply:
Oops, this was meant to be called "detailsWithoutErrors" to emphasize that an empty array seems to be returned rather than being null or absent.